$NetBSD: patch-am,v 1.1 2007/06/22 14:13:16 lkundrak Exp $

Fix for CVE-2007-3316 format-string vulnerability in Theora module described
by VideoLAN-SA-0702 advisory.  Backported from 0.8.6c.

--- modules/codec/theora.c.orig	2007-06-22 16:04:59.000000000 +0200
+++ modules/codec/theora.c
@@ -510,7 +510,7 @@ static void ParseTheoraComments( decoder
             *psz_value = '\0';
             psz_value++;
             input_Control( p_input, INPUT_ADD_INFO, _("Theora comment"),
-                           psz_name, psz_value );
+                           psz_name, "%s", psz_value );
         }
         free( psz_comment );
         i++;
